BASE/src/Microsoft.ApplicationInsights/DataContracts/EventTelemetry.cs

```diff
@@ -15,9 +15,9 @@
     /// </summary>
     public sealed class EventTelemetry : ITelemetry, ISupportProperties, ISupportAdvancedSampling, ISupportMetrics, IAiSerializableTelemetry
     {
-        internal const string TelemetryName = "Event";
-
+        internal const string DefaultEnvelopeName = "Event";
```

Comment:
This was the only event type that had previous internal const exposed to the pipeline. Preserving it as internal const for now - let me know if you'd like to refactor this.